playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -238,29 +238,22 @@
   any_errors_fatal: true
 
   pre_tasks:
+  - name: Load lib_openshift modules
```

Comment:
Just so I follow, this is done here since there is no ``meta/main.yml`` here correct?